### PR TITLE
re-added Joystick files that were removed by previous commit

### PR DESCRIPTION
**Pull request by MRW-Eric**
## Overview
Re-added the same files that were removed

## Changes
No changes to the files

## Verification
image

### CODE REVIEW

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -104,6 +104,7 @@ PlayerSettings:
   xboxOneMonoLoggingLevel: 0
   xboxOneLoggingLevel: 1
   xboxOneDisableEsram: 0
+  xboxOneEnableTypeOptimization: 0
```
Comment (RogPodge) [inline ProjectSettings/ProjectSettings.asset:107]: Don't include this file

File: ProjectSettings/QualitySettings.asset [hunk 2]
```diff
@@ -4,7 +4,7 @@
 QualitySettings:
   m_ObjectHideFlags: 0
   serializedVersion: 5
-  m_CurrentQuality: 5
+  m_CurrentQuality: 0
```
Comment (RogPodge) [inline ProjectSettings/QualitySettings.asset:7]: this one too

File: ProjectSettings/ProjectSettings.asset [hunk 3]
```diff
@@ -668,6 +670,8 @@ PlayerSettings:
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
     1: 
+    4: ARFOUNDATION_PRESENT
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset:673]: Looks like two settings files were included as well. Please remove these, as we don't want things like this scripting define checked-in.

**Pull request merged**